trpc/router/mailbox/issueGroups.ts

```
@@ -0,0 +1,407 @@
+    checkFeatureEnabled();
+
+    const userProfile = await db.query.userProfiles.findFirst({
+      where: eq(userProfiles.id, ctx.user.id),
+    });
+
+    if (!userProfile?.pinnedIssueGroupIds || userProfile.pinnedIssueGroupIds.length === 0) {
+      return { groups: [] };
+    }
+
+    // Convert to array of numbers if it's a string
+    const pinnedIds = Array.isArray(userProfile.pinnedIssueGroupIds) 
```

ESLint

[error] Delete `·`

(prettier/prettier)